CHANGELOG.rst

```diff
@@ -25,11 +61,16 @@ Internal Changes
 **Bug Fixes**
 
 - **Data services**: Handle spaces in ``provider_id`` for connected services (`#482 <https://github.com/SwissDataScienceCenter/renku-data-services/pull/482>`__).
+- **csi-rclone**: Do not log potentially sensitive data in error messages.
```

Comment:
Since we are updating csi-rclone the usual warning to admins should be issued here: to upgrade Renkulab when no sessions are mounting volumes with it.

Reply:
The changes for rclone were in the wrong version so I moved them to 0.60.0 and also I copied the same text warning admins about the unmounting as last time.